OpenGLViewSample/OpenGLViewSample/AppDelegate.cs

```diff
@@ -18,16 +18,17 @@ public AppDelegate ()
 		{
 		}
 
-		public override void DidFinishLaunching (NSNotification notification)
+		public override void DidFinishLaunching(NSNotification notification)
 		{
-			mainWindowController = new MainWindowController ();
-			view = new GLView (mainWindowController.Window.Frame, new NSOpenGLPixelFormat (new object [] {
+			mainWindowController = new MainWindowController();
+			Class.ThrowOnInitFailure = false;
```

Comment:
This sample fails to launch with the  error https://bugzilla.xamarin.com/attachment.cgi?id=23029, may be there is a different way to fix this issue. Otherwise, I will add a comment block.